lib/Profile.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:io';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:image_picker/image_picker.dart';
+
+class Profile extends StatefulWidget {
+  Profile({Key key}) : super(key: key);
+  @override
+  _ProfileState createState() => _ProfileState();
+}
+
+class _ProfileState extends State<Profile> {
+  File _image;
+
+  Future getImage() async {
+    var image = await ImagePicker.pickImage(source: ImageSource.gallery);
```

Comment:
Could you not use "var"? Try to use the actual type if you can find it.

Reply:
(oops sorry it's cause this was copy pasted from a tutorial 😬😬, fixed)